ArduPlane/Parameters.h

```diff
@@ -403,8 +403,10 @@ class Parameters {
     AP_Int8 waypoint_mode;
     AP_Int16 waypoint_radius;
     AP_Int16 waypoint_max_radius;
-    AP_Int16 loiter_radius;
     AP_Int16 rtl_radius;
+    
+    // Thermalling
+    AP_Int16 loiter_radius;
```

Comment:
I'm not sure what you mean... If this variable already existed for other flight modes then I wouldn't have been able to add it, but I was. The reason I added it is that for soaring it has a special meaning -- it's not the loiter radius of the "normal" loiter mode, but the radius for circling in a thermal.